extra/compose/default.bcccore-node.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
```
{
  "network": "livenet",
  "port": 3001,
  "services": [
    "explorer-api", // CHANGE TO THIS
    "bitcoind",
    "web"
  ],
  "servicesConfig": {
    "bitcoind": {     // IS THIS NEEDED?
      "spawn": {
        "datadir": "/data",
        "exec": "/usr/local/lib/node_modules/bcccore-node/bin/bitcoind"
      }
    },
    "explorer-api": {  // ADD THIS
      "module": "bcccore-explorer-api",
      "apiPrefix": "explorer-api"
    }
  }
}
```

Reply:
I added the explorer-api alias, but didn't change the bitcoin spawn config yet. Currently causes issues.

Reply:
ACK